src/Microsoft.VisualStudio.Threading.Analyzers/Utils.cs

```diff
@@ -101,6 +101,36 @@ internal static Action<OperationAnalysisContext> DebuggableWrapper(Action<Operat
             };
         }
 
+        internal static Action<OperationBlockStartAnalysisContext> DebuggableWrapper(Action<OperationBlockStartAnalysisContext> handler)
+        {
+            return ctxt =>
+            {
+                try
+                {
+                    handler(ctxt);
+                }
+                catch (OperationCanceledException)
+                {
+                    throw;
+                }
+                catch (Exception ex) when (LaunchDebuggerExceptionFilter())
+                {
+                    var messageBuilder = new StringBuilder();
+                    messageBuilder.AppendLine("Analyzer failure while processing syntax at");
```

Comment:
Do multi-line exception messages get printed alright at all the interesting points? (e.g. command line builds, VS error list, infobar, etc.)

Reply:
Honestly I don't know and I don't know all contexts I will need to test so I will move to a single-line message. From my tests it is ok with infobar and error list.